lib/Basics/win-utils.cpp

```diff
@@ -656,3 +727,29 @@ std::string getFileNameFromHandle(HANDLE fileHandle) {
   }
   return std::string((LPCTSTR)CString(FileInformation->FileName));
 }
+
+static std::vector<std::string> argVec;
+
+void TRI_GET_ARGV_WIN(int &argc, char** argv) {
+  auto wargStr = GetCommandLineW();
+
+  // if you want your argc in unicode, all you gonna do 
+  // is ask: 
+  wchar_t ** wargv = CommandLineToArgvW(wargStr, &argc);
+
+  argVec.reserve(argc);
+
+  UnicodeString buf;
```

Comment:
why are we using `UnicodeString` here instead of the same code as in rocksdb ?

Reply:
- I started out doing everything with ICU, even rocksdb.
- rocksdb doesn't have ICU
- rocksdb needs to maintain a two-fold solution, which can work well for other windows users without utf8-conversion.